ebuild-maintenance/git/text.xml

```diff
@@ -0,0 +1,322 @@
+<?xml version="1.0"?>
+<guide self="ebuild-maintenance/git/">
+<chapter>
+<title>Git for Gentoo Developers</title>
+
+<body>
+<p>
+This guide covers git usage instructions and policies specific to Gentoo ebuild
+development. It assumes that the readers possess basic git knowledge.
+For a generic guide, please see the official
+<uri link="https://git-scm.com/book/">git book</uri>.
+</p>
+
+
+<section>
+<title>Preparing a development checkout</title>
+<body>
+
+<subsection>
+<title>Cloning the gentoo.git repository</title>
+<body>
+
+<p>
+In order to commit or submit ebuild changes, you should work on top
+of the development git repository. In order to have push access, you should use
+the SSH protocol:
+</p>
+
+<pre>
+git clone git@git.gentoo.org:repo/gentoo.git
+</pre>
+
+<p>
+If you do not have SSH access to the Gentoo git service, you can use the anongit
+mirror for read-only access instead:
+</p>
+
+<pre>
+git clone https://anongit.gentoo.org/git/repo/gentoo.git
+</pre>
+
+<p>
+Normally git will fetch the complete history from the start of git repository
+(Aug 2015). This can require a significant amount of disk space. If you do not
+need the full history, you can use the <c>--depth</c> option to create a shallow
+clone, including only a subset containing the the newest commits. For example,
+<c>--depth=50</c> will include the 50 newest commits.
+</p>
+
+<p>
+Please note that git version 1.9 or newer is required to push when using
+a shallow clone.
+</p>
+
+</body>
+</subsection>
+
+<subsection>
+<title>Configuration specific to the Gentoo repository</title>
+<body>
+
+<p>
+To ensure that the Gentoo policies are followed, you should ensure to set
```

Comment:
Similarly, get rid of the repetitive `ensure`

Reply:
Done.